iphone/Classes/KrollCoverage.m

```diff
@@ -1,271 +0,0 @@
-/**
- * Appcelerator Titanium Mobile
```

Comment:
Is this file no more used?

Reply:
Yes, it was not used since ever, so I removed it for now.